android/app/src/main/java/org/jitsi/meet/GoogleServicesHelper.java

```diff
@@ -0,0 +1,34 @@
+package org.jitsi.meet;
+
+import android.net.Uri;
+import android.util.Log;
+
+import com.crashlytics.android.Crashlytics;
+import com.google.firebase.dynamiclinks.FirebaseDynamicLinks;
+import io.fabric.sdk.android.Fabric;
+
+import org.jitsi.meet.sdk.JitsiMeetActivity;
+
+
+final class GoogleServicesHelper {
```

Comment:
Some explanation on what is happening in the docs for this class would be useful